packages/p2p-v2/src/features/my-profile/screens/MyProfileStats/MyProfileStats.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import { useActiveAccount } from '@deriv/api';
+import { useAdvertiserStats } from '../../../../hooks';
+import './MyProfileStats.scss';
+
+type MyProfileStatsItemProps = {
+    currency?: string;
+    duration?: string;
+    label: string;
+    onClickLifetime?: (isLifetimeClicked: boolean) => void;
+    shouldShowLifetime?: boolean;
+    value: string;
+};
+const MyProfileStatsItem = ({
```

Comment:
Can we move this into a separate file? keeping this here makes the component too big so later it would be hard to debug this